src/main.ts

```diff
@@ -0,0 +1,12 @@
+import { MRAIDImplementation } from "./mraid";
+import { MRAID } from "./mraidapi";
+
+export {};
+
+declare global {
+  interface Window {
+    mraid: MRAID;
+  }
+}
+
+window.mraid = MRAIDImplementation.create();
```

Comment:
Shouldn't you check before if MRAID property is not already set ? Like by doing: `window.mraid = window.mraid ?? MRAIDImpl.create()`.

Reply:
Sure, will change that